java/languages.antlr/src/org/netbeans/modules/languages/antlr/refactoring/Refactoring.java

```diff
@@ -148,7 +148,7 @@ public Problem prepare(RefactoringElementsBag refactoringElements) {
                         if(ref.defOffset != null) {
                             ranges.add(ref.defOffset);
                         }
-                        ranges.addAll(ref.occurances);
+                        ranges.addAll(result.getOccurrences(name));
```

Comment:
This needs to be pulled out of the if. I tested this:

- I opened `LexBasic.g4` from the `antlr4` package
- I ran "Find Usage" in `Ws`

I only got the results from `LexBasic.g4` and was missing the occurrences from `ANTLR4Lexer.g4`. `ref` is `NULL` for the latter, but there are still occurrences. Pulling out this call fixes it.